src/serac/infrastructure/tests/error_handling.cpp

```diff
@@ -38,10 +38,9 @@ TEST(ErrorHandling, BcOneComponentVectorCoef)
 
   auto mesh = mesh::refineAndDistribute(buildDiskMesh(10), 0, 0);
 
-  mfem::H1_FECollection       coll(1, mesh->Dimension());
-  mfem::ParFiniteElementSpace space(mesh.get(), &coll);
+  auto [space, coll] = serac::generateParFiniteElementSpace<H1<1>>(mesh.get());
 
-  EXPECT_THROW(BoundaryCondition(coef, 0, space, std::set<int>{1}), SlicErrorException);
+  EXPECT_THROW(BoundaryCondition(coef, 0, *space.get(), std::set<int>{1}), SlicErrorException);
```

Comment:
Zach, I expect your C++ usage is more fluent than mine, but to my eye, the use of the dereference with the dot looks strange instead of just using the `->` operator.

Reply:
I agree, it's because we have a `std::unique_ptr<mfem::FiniteElementSpace> space` and we need a `const mfem::FiniteElementSpace&`, so `space.get()` returns the raw pointer and `*(space.get())` returns the dereferenced object. Note: that's not actually doing the same thing as `->`, the dereference happens second. It's kinda clunky, I'm open to suggestions of how to improve clarity.

Reply:
Jk, apparently that's what `operator*` does. Updating now